<launch>

  <arg name="use_monitor" default="true"/>
  <arg name="use_collision_map" default="true"/>

  <node pkg="planning_environment" type="environment_server" output="screen" name="environment_server">      
    <param name="use_monitor" type="bool" value="$(arg use_monitor)"/>
    <param name="use_collision_map" type="bool" value="$(arg use_collision_map)"/>
    <!-- <remap from="/joint_states" to="/arm_1/joint_states"/> -->
  </node>

  <node pkg="planning_environment" type="planning_scene_validity_server" output="screen" name="planning_scene_validity_server"/>      

</launch>
